index.html

```diff
@@ -852,11 +852,21 @@ <h2><dfn>CaptureController</dfn></h2>
           </p>
           <pre class="idl">
             [Exposed=Window, SecureContext]
-            interface CaptureController {
+            interface CaptureController : EventTarget {
               constructor();
               undefined setFocusBehavior(CaptureStartFocusBehavior focusBehavior);
             };
           </pre>
+          <div class="issue atrisk">
+            <p>
+              {{CaptureController}} does not yet not define any event handlers.
+              As such, it is not required to inherit from {{EventTarget}}.
+              This inheritance was added for the benefit of other specifications
```

Comment:
```suggestion
              This inheritance is added for the benefit of other specifications
```

Reply:
Sounds slightly odd to me, as the text will still be there, in the present tense, long after we make this decision. Let's run it by Bernard during the editors' meeting and take what he prefers.

Reply:
cc @aboba